docs/src/pages/index.js

```diff
@@ -137,60 +137,64 @@ const Home = ({ sponsors }) => {
               </div>
             </div>
           </div>
-          <div className="py-6">
-            <div className="uppercase tracking-wider text-sm font-semibold text-center text-gray-400 mb-3">
-              Trusted in Production by
-            </div>
+          <div className="bg-gray-900 text-white">
+            <div className="py-32">
+              <div className="mx-auto container text-center">
+                <div className="text-xl">THE OFFICIAL COURSE</div>
+                <div className="text-3xl md:text-5xl font-bold">
+                  React Query Essentials
```

Comment:
Do I need to change the name of the course to "React Query Essentials" on the course platform?

Reply:
Updated